frontend/src/scenes/feature-flags/FeatureManagementScenes.tsx

```diff
@@ -0,0 +1,63 @@
+import { Spinner } from 'lib/lemon-ui/Spinner'
+import { lazy, Suspense } from 'react'
+
+const EarlyAccessFeatures = lazy(() =>
```

Comment:
note for reviewing later: these lazy imports are working well in local dev, but it is unprecedented in the app. Check to make sure there isn't some reason for not using it before. If so, it's a superfluous optimization anyway, but nice for speedy loads